content/docs/command-reference/exp/run.md

```diff
@@ -52,8 +52,8 @@ out by DVC). Note that these commits are not pushed to the Git remote by default
 
 </details>
 
-The results of the last experiment can be seen in the <abbr>workspace</abbr>. To
-display and compare your experiments, use `dvc exp show` or `dvc exp diff`. Use
+The results of the last experiment can be seen in the workspace. To display and
+compare your experiments, use `dvc exp show` or `dvc exp diff`. Use
```

Comment:
Maybe we should document that all of the other DVC diff commands and plots commands can also be used with experiments here

Reply:
I'd wait to see if all the `show` and `diff` commands get merged 🙂

Reply:
Unification of the `show` and `diff` commands is something that probably won't happen for a decent amount of time, and I brought it up because I have seen questions from users in discord asking about how to use plots with experiments.

Reply:
Oh. I thought you meant `metrics/params diff`, which are mentioned in https://dvc.org/doc/command-reference/exp/diff. I see you also mentioned `plots` now, and I do agree that should be mentioned but maybe also from the `exp diff` ref. for now? ⌛

Reply:
> DVC diff commands and plots commands can also be used with experiments here

Wait. But we mean more like show and compare MANY/ALL experiments here. `metrics/params diff` only take 2 and in those cases `exp diff` is more obvious anyway.

This  only really applies to `plots diff` as of now, I think? Mentioned just that in 29004e6b for now.

Reply:
I think that approach makes sense @jorgeorpinel. Alternatively, we could say something like "any dvc command that accepts Git revisions also accepts experiment names."